JS.md

```diff
@@ -205,6 +205,25 @@ to the Object `{ module: String(i.module_name), name: String(i.item_name), kind:
 
 Note: other fields like `signature` may be added in the future.
 
+### `WebAssembly.Module.userSections`
```

Comment:
We call them "unknown" sections in the binary format. I'd rather be consistent (either name is fine).